pkg/auth/handler/webapp/authflow_forgot_password.go

```diff
@@ -226,7 +225,6 @@ func (h *AuthflowForgotPasswordHandler) ServeHTTP(w http.ResponseWriter, r *http
 
 	h.Controller.HandleStartOfFlow(w, r, webapp.SessionOptions{}, authflow.FlowReference{
 		Type: authflow.FlowTypeAccountRecovery,
-		Name: flowName,
```

Comment:
To avoid confusion, let's change all argument type from FlowReference to FlowType. It is because the name is now derived by the controller. It eliminates the possibility that someone later on passes a name and thinks name is not respected.